packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -46,6 +50,10 @@ export interface SnapWPConfig {
 	 * html-react-parser overload options
 	 */
 	parserOptions?: HTMLReactParserOptions;
+	/**
+	 * Sitemap configuration.
+	 */
+	sitemapConfig?: SitemapConfig;
```

Comment:
`*Config` is redundant this whole this is a config object.
```suggestion
	sitemap?: SitemapConfig;
```